src/api/participantsRouter.ts

```diff
@@ -253,7 +253,15 @@ export function createParticipantsRouter() {
         ...users.map((user) =>
           assignClientRoleToUser(kcAdminClient, user.email, 'api-participant-member')
         ),
-        participant!.$query().upsertGraph(data),
+        Participant.query().upsertGraph(
+          {
+            id: participant!.id!,
+            ...data,
+          },
+          {
+            relate: true,
```

Comment:
This will update the many to one relationship between participant and participant types